src/WebJobs.Extensions.DurableTask/LocalHttpListener.cs

```diff
@@ -36,27 +36,16 @@ internal class LocalHttpListener : IDisposable
             this.handler = handler ?? throw new ArgumentNullException(nameof(handler));
             this.durableTaskOptions = durableTaskOptions ?? throw new ArgumentNullException(nameof(durableTaskOptions));
 
-#if !FUNCTIONS_V1
-            this.InternalRpcUri = new Uri($"http://127.0.0.1:{this.GetAvailablePort()}/durabletask/");
-            var listenUri = new Uri(this.InternalRpcUri.GetLeftPart(UriPartial.Authority));
-            this.localWebHost = new WebHostBuilder()
-                .UseKestrel()
-                .UseUrls(listenUri.OriginalString)
-                .Configure(a => a.Run(this.HandleRequestAsync))
-                .Build();
-#else
-            // Just use default port for internal Uri. No need to check for port availability since
-            // we won't be listening to this endpoint.
-            this.InternalRpcUri = new Uri($"http://127.0.0.1:{DefaultPort}/durabletask/");
+            // Set to a non null value
+            this.InternalRpcUri = new Uri($"http://127.0.0.1:{DefaultPort}/defaultvalue/");
             this.localWebHost = new NoOpWebHost();
-#endif
         }
 
-        public Uri InternalRpcUri { get; }
+        public Uri InternalRpcUri { get; private set; }
 
         public bool IsListening { get; private set; }
 
-        public void Dispose() => this.localWebHost.Dispose();
+        public void Dispose() => this.localWebHost?.Dispose();
```

Comment:
It doesn't look like `this.localWebHost` can ever be since we assign it in the constructor. We also never explicitly assign it to null. Is this a leftover artifact of an earlier iteration?

Reply:
Indeed. Good catch.